.golangci.yml

```diff
@@ -1,16 +1,30 @@
 linters:
   enable:
+  - wastedassign
```

Comment:
Would it be more readable to sort alphabetically the linters?

Reply:
Taken care of in https://github.com/Kong/deck/pull/402